features/main/issue_5736_aerendir.feature

```diff
@@ -0,0 +1,237 @@
+  Scenario: PUT Team with PUT Employee
+    Given I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/issue5736_companies/" with body:
+    """
+    {
+      "name": "Company 1"
+    }
+    """
+
+    And I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/issue5736_companies/1/issue5736_teams" with body:
+    """
+    {
+      "name": "Team 1",
+      "employees": [
+        {
+          "name": "Employee 1"
+        },
+        {
+          "name": "Employee 2"
+        },
+        {
+          "name": "Employee 3"
+        }
+      ]
+    }
+    """
+
+    And I add "Content-Type" header equal to "application/json"
+    And I send a "PUT" request to "/issue5736_companies/1/issue5736_teams/1" with body:
+    """
+    {
+      "name": "Team 1",
+      "employees": [
+        {
+          "@id": "/issue5736_companies/1/issue5736_teams/1/issue5736_employees/1",
+          "name": "Employee 1"
+        },
+        {
+          "@id": "/issue5736_companies/1/issue5736_teams/1/issue5736_employees/2",
+          "name": "Employee 2"
+        },
+        {
+          "@id": "/issue5736_companies/1/issue5736_teams/1/issue5736_employees/3",
+          "name": "Employee 3 edited"
+        }
+      ]
+    }
+    """
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the JSON should be equal to:
+    """
+    {
+      "@id": "/issue5736_companies/1/issue5736_teams/1",
+      "@type": "Issue5736Team",
+      "name": "Team 1",
+      "employees": [
+        {
+          "@id": "/issue5736_companies/1/issue5736_teams/1/issue5736_employees/1",
+          "@type": "Issue5736Employee",
+          "name": "Employee 1"
+        },
+        {
+          "@id": "/issue5736_companies/1/issue5736_teams/1/issue5736_employees/2",
+          "@type": "Issue5736Employee",
+          "name": "Employee 2"
+        },
+        {
+          "@id": "/issue5736_companies/1/issue5736_teams/1/issue5736_employees/3",
+          "@type": "Issue5736Employee",
+          "name": "Employee 3 edited"
+        }
+      ]
+    }
+    """
+
+  @createSchema
+  Scenario: PUT Team with DELETE Employee
+    Given I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/issue5736_companies/" with body:
+    """
+    {
+      "name": "Company 1"
+    }
+    """
+
+    And I add "Content-Type" header equal to "application/json"
+    And I send a "POST" request to "/issue5736_companies/1/issue5736_teams" with body:
+    """
+    {
+      "name": "Team 1",
+      "employees": [
+        {
+          "name": "Employee 1"
+        },
+        {
+          "name": "Employee 2"
+        },
+        {
+          "name": "Employee 3"
+        }
+      ]
+    }
+    """
+
+    And I add "Content-Type" header equal to "application/json"
+    And I send a "PUT" request to "/issue5736_companies/1/issue5736_teams/1" with body:
+    """
```

Comment:
okay so I actually don't know how this works :laughing: 

I'll take a look at that! Thanks for the reproducer!